src/species_advance/standard/advance_p.cc

```diff
@@ -87,7 +87,13 @@ advance_p_pipeline( advance_p_pipeline_args_t * args,
     ux  += hax;                               // Half advance E
     uy  += hay;
     uz  += haz;
+
+#ifdef ENABLE_NON_RELATIVSTIC
+    v0 = 1.0f;
+#else
     v0   = qdt_2mc/sqrtf(one + (ux*ux + (uy*uy + uz*uz)));
```

Comment:
Do we also need to change the `v0   = one/sqrtf(one + (ux*ux+ (uy*uy + uz*uz)));` calculations?